tests/integration/zenodo_test.py

```diff
@@ -0,0 +1,212 @@
+"""Test zenodo client api."""
```

Comment:
This test occasionally raises an exception from line 284 of `api_client.py`:

`Error Zenodo should only return a single deposition`

This error does not occur all of the time however, and should be investigated further.

Reply:
We fixed this - the culprit was:
1. we published a deposition and, soon after, created a new version
2. When we tried to get the deposition for the concept DOI, the old published version and the new unpublished one were both returned
3. We chose to pick the newer one (based on the DOI monotonically increasing)